### PR TITLE
Add amqp1 plugin support

### PR DESCRIPTION
**Pull request by paramite**
#### Pull Request (PR) description
This patch adds possibility to configure amqp1 plugin added recently
to collectd [1].

[1] https://github.com/collectd/collectd/pull/2618

### CODE REVIEW

File: spec/classes/collectd_plugin_amqp1_spec.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                  graphite_escape_char: '_',
+                  graphite_separate_instances: false,
+                  graphite_always_append_ds: false,
+                  graphite_preserve_separator: false
+                }
+            ]
+          }
+        end
+
+        it { is_expected.to contain_file('amqp1.load').with(content: %r{<Transport "transport">}) }
```
Comment (bastelfreak) [inline spec/classes/collectd_plugin_amqp1_spec.rb]: can you add a `it { is_expected.to compile.with_all_deps }`

File: templates/plugin/amqp1.conf.erb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<Plugin amqp1>
```
Comment (bastelfreak) [inline templates/plugin/amqp1.conf.erb]: can you please provide this as an epp and not erb template? We agreed on epp some time ago and documented it in our review guideline at https://voxpupuli.org/docs/#reviewing-a-module-pr

File: spec/classes/collectd_plugin_amqp1_spec.rb (hunk 1 above)
Comment (paramite) [reply spec/classes/collectd_plugin_amqp1_spec.rb]: Sure will do.

File: templates/plugin/amqp1.conf.erb (hunk 2 above)
Comment (paramite) [reply templates/plugin/amqp1.conf.erb]: Ok, didn't notice this as all templates are in ERB. Will rewrite the template then.

**Comment by bastelfreak** [comment]
@paramite can you take a look at the failing travis jobs?

**Comment by paramite** [comment]
Yes, I'mrreally trying, but EPP and syntax check is giving me hard times :).

**Comment by paramite** [comment]
@bastelfreak Done.

**Pull request merged**